risc0/circuit/rv32im/src/execute/executor.rs

```diff
@@ -309,6 +322,50 @@ impl<'a, 'b, S: Syscall> Executor<'a, 'b, S> {
         })
     }
 
+    fn dump_segment(
+        &mut self,
+        po2: usize,
+        segment_threshold: u32,
+        index: u64,
+    ) -> anyhow::Result<()> {
+        if let Some(dump_path) = std::env::var_os("RISC0_DUMP_PATH") {
+            tracing::error!(
+                "Execution failure, saving segment to {}:",
+                dump_path.to_string_lossy()
```

Comment:
its better to use `.display()` in this situation

Reply:
Actually, this is an `OsString` and that doesn't have `display()`

Reply:
Whoops my mistake